packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page/definition-tab/property-list-card/shared/property-type-form/data-type-selector/custom-expected-value-builder/array-expected-value-builder.tsx

```diff
@@ -28,11 +14,13 @@ import { dataTypeOptions as primitiveDataTypeOptions } from "../../shared/data-t
 import { expectedValuesOptions } from "../shared/expected-values-options";
 import { ExpectedValueBadge } from "./array-expected-value-builder/expected-value-badge";
 import { DeleteExpectedValueModal } from "./array-expected-value-builder/delete-expected-value-modal";
+import { CustomExpectedValueSelector } from "./shared/custom-expected-value-selector";
+import { ObjectExpectedValueBuilder } from "./object-expected-value-builder";
```

Comment:
I *think* this import is technically not allowed under the import rules and both files should be in a `shared` folder, but @kachkaev will be able to confirm

Reply:
Addressed in 6c585f48b8fb214452f5ebad112232d30b2da205. Thank you

Reply:
We spoke about a couple of other instances of new imports violating the rules that Luís is going to fix